presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveQueries.java

```diff
@@ -571,17 +571,32 @@ public void testCreateTableAsSelect()
                 .setCatalogSessionProperty("hive", "collect_column_statistics_on_write", "false")
                 .build();
 
-        getQueryRunner().execute(session, "CREATE TABLE tmp AS SELECT * FROM nation");
-        assertQuery("SELECT * FROM tmp", "SELECT * FROM nation");
+        // Clean up if table tmp already exists.
         dropTable("tmp");
```

Comment:
Nit: In my mind renaming it to "dropTableIfExists()" is more self-explanatory.

Reply:
Fixed.